CMakeLists.txt

```diff
@@ -185,12 +185,14 @@ if(NOT COMMAND find_host_program)
 endif()
 
 # Tests require Python3
-find_host_package(PythonInterp 3 REQUIRED)
+if(NOT SPIRV_SKIP_TESTS)
```

Comment:
Python is used to build the grammar tables.  The reason this works even if python is used is because `${PYTHON_EXECUTABLE}` expands to the empty string, and the python scripts in `utils` are all have execute permission.  When they run, they just look for `python` on your path.

We should be searching for python in all cases.  If there is a fix needed, it is to change the permission on those files.

Reply:
This originally came up because I tried to get it running on an automated build system where it didn't have Python 3 available and it appeared to not be required. I missed the requirement for the tables, and it must have just worked for Python 2.

I'll revert these changes and just have the export/install fixes then.